services/src/util/tests.rs

```diff
@@ -124,6 +129,31 @@ pub async fn register_workflow_helper<C: Context>(ctx: &C) -> (Workflow, Workflo
     (workflow, id)
 }
 
+pub async fn add_ndvi_to_datasets(ctx: &InMemoryContext) -> DataSetId {
```

Comment:
i think there is already a method with the same/similar name?

Reply:
there ist `create_` which just creates the metadata and `add_` which creates the metadata and adds it to the context